Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -7711,8 +7712,7 @@ def execute_command(command, args, extract_contents=True, fail_on_error=True):
             return res
         else:
             return True, res
-
-    contents = [entry.get('Contents', {}) for entry in res]
+    contents = [entry.get('Contents', {}) for entry in res if entry['Type'] != entryTypes['debug']]
```

Comment:
I think you should remove it only if its contents is empty.
and also pleas explain in the PR details why this is bing done.(that version 8 is returning a dbug entry with empty contents that breaks BC)

Reply:
from what I understand the contents in debug-entry is always empty, the debug entry contain the debug_file location in the file key.